### PR TITLE
High Flow Next support

### LINKED ISSUE #8: Adding support for High Flow Next
Would it be possible to add support for the High Flow Next flow meter?

https://github.com/medevil84/FanControl.AquacomputerHighFlowNext might be helpful.

### PR DESCRIPTION
**Pull request by aleksamagicka**
Add support for useful High Flow Next sensors.

Closes #8.

### CODE REVIEW

**Review by leoratte — COMMENTED** [review]
I don't think the high flow sensors should be in `num_fans`. maybe we can add something like `num_misc_sensors` for this and and also for the other special cases currently implemented.

File: aquacomputer_d5next.c [hunk 1]
```diff
@@ -197,6 +208,27 @@ static const char *const label_fan_flow_speed_quadro[] = {
...
 };
 
+/* Labels for High Flow Next */
+static const char *const label_highflownext_temp_sensors[] = {
+	"Coolant temp",
+	"External sensor"
+};
+
+static const char *const label_highflownext_fan_speed[] = {
+	"Calibrated flow [dL/h]",
```
[the remark this answers is not in the document]
Comment (aleksamagicka) [reply aquacomputer_d5next.c:218]: As far as I've gleamed from the hwinfo64 picture and sensors output in #8, HFN gives readings with one decimal. On the picture, it's 126.5, and from the output it seems to be 93.5 ~ 94 when rounded up. But when we read it from the report, it gives 935, so it's denoted as dL/h.

Of course, if I'm wrong, we'll fix it, but for now that seems like a correct assumption. We should be using whichever corresponds to what the device is giving. I've found this image on the hwinfo64 forums just now:



Looks like it also reports with one decimal place. The Quadro code now divides the flow value by 10, so can we strip that and report in dL/h?

**Comment by aleksamagicka** [comment]
I see what you're saying and I agree, but `num_misc_sensors` seems too general and is device dependent anyway. I think it's a better idea to just move the three "fans" the HFN has into the special cases switch, and let the temperature sensors be read normally.

File: aquacomputer_d5next.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (leoratte) [reply aquacomputer_d5next.c:218]: Sure, I see no problem in changing the quadro code.

[the remark this answers is not in the document]
Comment (aleksamagicka) [reply aquacomputer_d5next.c:218]: Thanks, pushed a commit just now.

**Review by leoratte — COMMENTED** [review]
With` num_fans` now being 0 for HFN, we need a new case for HFN in `aqc_is_visible` in `hwmon_fan` to make the make the three HFN sensors visible again.

**Comment by aleksamagicka** [comment]
Thanks, missed that yesterday evening - that should be resolved now.

**Pull request merged**